modules/tinymce/src/plugins/emoticons/test/ts/browser/EmoticonTwemojiTest.ts

```diff
@@ -0,0 +1,66 @@
+import { Assertions, Chain, FocusTools, Keyboard, Keys, Log, Pipeline, UiFinder, Waiter } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import { Attribute, SugarBody, SugarDocument, SugarElement } from '@ephox/sugar';
+
+import EmoticonsPlugin from 'tinymce/plugins/emoticons/Plugin';
+import SilverTheme from 'tinymce/themes/silver/Theme';
+import { cFakeEvent } from '../module/test/Utils';
+
+UnitTest.asynctest('browser.tinymce.plugins.emoticons.SearchTest', (success, failure) => {
+  EmoticonsPlugin();
+  SilverTheme();
+
+  TinyLoader.setupLight(function (editor, onSuccess, onFailure) {
+    const tinyApis = TinyApis(editor);
+    const tinyUi = TinyUi(editor);
+    const doc = SugarDocument.getDocument();
+
+    Pipeline.async({},
+      Log.steps('TBA', 'Emoticons: Open dialog, Search for "dog", Dog should be first option', [
+        tinyApis.sFocus(),
+        tinyUi.sClickOnToolbar('click emoticons', 'button'),
+        Chain.asStep({}, [
+          tinyUi.cWaitForPopup('wait for popup', 'div[role="dialog"]')
```

Comment:
Why not `sWaitForPopup`

Reply:
This is a direct copy of another test where I just changed the setting/assertion, so it seems to have the same problem.

Reply:
Fixed